### PR TITLE
chore: Bumpenvs for NGC+ images

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Bumpenvs for NGC+ Images (and tf2.8 removal)
https://github.com/determined-ai/environments/pull/235

## Test Plan

CI doesn't fail

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
MD-270

### CODE REVIEW

**Review by ioga — APPROVED** [review]

**Review by keita-determined — APPROVED** [review]

File: docs/release-notes/ngc-images.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+-  Add Add early access NVIDIA NGC-based images to our environments offerings. These images can be
+   pulled from `https://hub.docker.com/r/determinedai/pytorch-ngc` or
+   `https://hub.docker.com/r/determinedai/tensorflow-ngc`. By downloading and using these images,
+   you accept the terms and conditions of all third-party software licenses contained within,
+   including the (`NVIDIA Deep Learning Container
+   License.<https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`_)
+   Users can build their own images from a given NGC container version by using the
+   `build-pytorch-ngc` or `build-tensorflow-ngc` workflows found in our environments `MakeFile`. Our
+   environments repo is located at `https://github.com/determined-ai/environments`.
+
```
Comment (tara-det-ai) [inline docs/release-notes/ngc-images.rst:14]: :orphan:

**New Features**

-  Include early-access NVIDIA NGC-based images in our environment offerings. These images are accessible from `https://hub.docker.com/r/determinedai/pytorch-ngc` or
   `https://hub.docker.com/r/determinedai/tensorflow-ngc`. By downloading and using these images, users acknowledge and agree to the terms and conditions of all third-party software licenses contained within, including the `NVIDIA Deep Learning Container License
<https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`__.
   Users can build their own images from a specified NGC container version by using the
   ``build-pytorch-ngc`` or ``build-tensorflow-ngc`` workflows located in our environments ``MakeFile``. The environments repository is located at `https://github.com/determined-ai/environments`.

Comment (tara-det-ai) [reply docs/release-notes/ngc-images.rst:14]: wow, it's nearly impossible to show "restructured text markup" in a "markdown" comment

Comment (tara-det-ai) [reply docs/release-notes/ngc-images.rst:14]: `:orphan:

**New Features**

-  Include early-access NVIDIA NGC-based images in our environment offerings. These images are
   accessible from `pytorch-ngc <https://hub.docker.com/r/determinedai/pytorch-ngc>`_ or
   `tensorflow-ngc <https://hub.docker.com/r/determinedai/tensorflow-ngc>`_. By downloading and
   using these images, users acknowledge and agree to the terms and conditions of all third-party
   software licenses contained within, including the `NVIDIA Deep Learning Container License
   <https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`__.
   Users can build their own images from a specified NGC container version by using the
   ``build-pytorch-ngc`` or ``build-tensorflow-ngc`` workflows located in our environments
   ``MakeFile`` in the `environments repository <https://github.com/determined-ai/environments>`_.
`

Comment (tara-det-ai) [reply docs/release-notes/ngc-images.rst:14]: applied a commit that fixes the formatting and modifies some word choices

**Review by hkang1 — APPROVED** [review]
